crates/epoch-snark/src/lib.rs

```diff
@@ -30,14 +30,13 @@ fn convert_result_to_bool<T, E: Display, F: Fn() -> Result<T, E>>(f: F) -> bool
 pub extern "C" fn encode_epoch_block_to_bytes(
     in_epoch_index: c_ushort,
     in_maximum_non_signers: c_uint,
-    in_aggregated_public_key: *const PublicKey,
     in_added_public_keys: *const *const PublicKey,
     in_added_public_keys_len: c_int,
+    in_should_encode_aggregated_pk: bool,
```

Comment:
As in my other comments, I think it'd be better if this were a separate function. Not a dealbreaker though.

Reply:
For this one I think I'll keep it as is because of the FFI complexity...